diff --git a/node_modules/@tamagui/element/dist/cjs/getWebElement.js b/node_modules/@tamagui/element/dist/cjs/getWebElement.js
index 50068c7..5686e47 100644
--- a/node_modules/@tamagui/element/dist/cjs/getWebElement.js
+++ b/node_modules/@tamagui/element/dist/cjs/getWebElement.js
@@ -20,7 +20,7 @@ module.exports = __toCommonJS(getWebElement_exports);
 function getWebElement(element) {
   if (!element)
     throw new Error("Element is null or undefined");
-  if (!(element instanceof HTMLElement))
+  if (typeof element !== 'object' || !("nodeType" in element))
     throw new Error("Element is not an HTMLElement");
   return element;
 }
diff --git a/node_modules/@tamagui/element/dist/esm/getWebElement.js b/node_modules/@tamagui/element/dist/esm/getWebElement.js
index 56d9350..c390f09 100644
--- a/node_modules/@tamagui/element/dist/esm/getWebElement.js
+++ b/node_modules/@tamagui/element/dist/esm/getWebElement.js
@@ -1,7 +1,7 @@
 function getWebElement(element) {
   if (!element)
     throw new Error("Element is null or undefined");
-  if (!(element instanceof HTMLElement))
+  if (typeof element !== 'object' || !("nodeType" in element))
     throw new Error("Element is not an HTMLElement");
   return element;
 }
diff --git a/node_modules/@tamagui/element/dist/esm/getWebElement.mjs b/node_modules/@tamagui/element/dist/esm/getWebElement.mjs
index dfd702e..a619314 100644
--- a/node_modules/@tamagui/element/dist/esm/getWebElement.mjs
+++ b/node_modules/@tamagui/element/dist/esm/getWebElement.mjs
@@ -1,6 +1,6 @@
 function getWebElement(element) {
   if (!element) throw new Error("Element is null or undefined");
-  if (!(element instanceof HTMLElement)) throw new Error("Element is not an HTMLElement");
+  if (typeof element !== 'object' || !("nodeType" in element)) throw new Error("Element is not an HTMLElement");
   return element;
 }
 export { getWebElement };
diff --git a/node_modules/@tamagui/element/dist/esm/getWebElement.native.js b/node_modules/@tamagui/element/dist/esm/getWebElement.native.js
index 46dc615..134f378 100644
--- a/node_modules/@tamagui/element/dist/esm/getWebElement.native.js
+++ b/node_modules/@tamagui/element/dist/esm/getWebElement.native.js
@@ -1,9 +1,6 @@
-function _instanceof(left, right) {
-  return right != null && typeof Symbol < "u" && right[Symbol.hasInstance] ? !!right[Symbol.hasInstance](left) : left instanceof right;
-}
 function getWebElement(element) {
   if (!element) throw new Error("Element is null or undefined");
-  if (!_instanceof(element, HTMLElement)) throw new Error("Element is not an HTMLElement");
+  if (typeof element !== 'object' || !("nodeType" in element)) throw new Error("Element is not an HTMLElement");
   return element;
 }
 export { getWebElement };
diff --git a/node_modules/@tamagui/element/src/getWebElement.ts b/node_modules/@tamagui/element/src/getWebElement.ts
index f32c0c7..50b8000 100644
--- a/node_modules/@tamagui/element/src/getWebElement.ts
+++ b/node_modules/@tamagui/element/src/getWebElement.ts
@@ -16,7 +16,7 @@ export function getWebElement<T extends HTMLElement = HTMLElement>(
   if (!element) {
     throw new Error('Element is null or undefined')
   }
-  if (!(element instanceof HTMLElement)) {
+  if (typeof element !== 'object' || !('nodeType' in element)) {
     throw new Error('Element is not an HTMLElement')
   }
   return element as unknown as T
