diff --git a/node_modules/react-native-nitro-player/ios/core/TrackPlayerCore.swift b/node_modules/react-native-nitro-player/ios/core/TrackPlayerCore.swift
index 7d57df3..3b684ab 100644
--- a/node_modules/react-native-nitro-player/ios/core/TrackPlayerCore.swift
+++ b/node_modules/react-native-nitro-player/ios/core/TrackPlayerCore.swift
@@ -630,8 +630,6 @@ class TrackPlayerCore: NSObject {
       self.updatePlayerQueue(tracks: playlist.tracks)
       // Emit initial state (paused/stopped before play)
       self.emitStateChange()
-      // Automatically start playback after loading
-      self.play()
     } else {
       print("   ❌ Playlist NOT FOUND")
       print(String(repeating: "🎼", count: Constants.playlistSeparatorLength) + "\n")
